uint16 obs_dist
float32 obs_direct
float32 goal_x
float32 goal_y
int32 obs_latelitude